### ISSUE TITLE
In!

### ISSUE THREAD

**Issue by hugovk** [NaNoGenMo/2018#27]
I'm going to start off with something easyish, and then see what follows.

---

First by reproducing some historical procgen shared by [James Ryan](https://twitter.com/xfoml/status/885584010411253761):

> extremely early #procgen: Pfizer used an IBM 702 in 1956 to generate 42,000 prospective drug names, which were compiled into a printed book!

![image](https://user-images.githubusercontent.com/1324225/47817585-b64d8c80-dd5e-11e8-93ff-f256176574c6.png)
![image](https://user-images.githubusercontent.com/1324225/47817607-c2d1e500-dd5e-11e8-9748-58c86fb56154.png)![image](https://user-images.githubusercontent.com/1324225/47817612-c6656c00-dd5e-11e8-8a03-3849e96e718b.png)

I'll start with one or both of these from [Corpora](https://github.com/dariusk/corpora) to get word endings:
* https://github.com/dariusk/corpora/blob/master/data/medicine/drugs.json
* https://github.com/dariusk/corpora/blob/master/data/medicine/drugNameStems.json

I'll find a bunch of one and two syllable words, then combine them to get ~42,000 words.

![image](https://user-images.githubusercontent.com/1324225/47817623-cbc2b680-dd5e-11e8-86f8-4a25721e9c53.png)

Only 42k words? Worry not.

Like the IBM machine, it will be "taught to be discreet" and "fixed to automatically eliminate four-letter combinations that wouldn't be proper in a family medicine chest". So I'll need a list of "improper words" to filter those out. And unlike the original, I think they will belong in an ~8kword appendix.

---

I may also dip into @ojahnn's [NaNoLiPo](https://github.com/ojahnn/NaNoLiPo2018) for other ideas, or some unrealised ideas from [2017](https://github.com/NaNoGenMo/2017/issues/79), [2016](https://github.com/NaNoGenMo/2016/issues/20) and [2015](https://github.com/dariusk/NaNoGenMo-2015/issues/8) and [2014](https://github.com/dariusk/NaNoGenMo-2014/issues/25).

**Comment by hugovk** [comment]
# NaNoLiPo Prompts
* [x] Nov. 1: **#avoidlipo** -- Your text may not contain any "e"s
  https://github.com/NaNoGenMo/2018/issues/30
* [ ] Nov. 2: **#monosyllablipo** -- All words in your text must consist of exactly one syllable.
* [ ] Nov. 3: **#punchcardlipo** -- Each sentence in your text must contain [exactly 80 characters](https://en.wikipedia.org/wiki/Punched_card#IBM_80-column_punched_card_format_and_character_codes).
* [ ] Nov. 4: **#tautogramlipo** -- All words in your text must [begin with the same letter](https://en.wikipedia.org/wiki/Tautogram). You can choose the letter.
* [ ] Nov. 5: **#vomitlipo** -- In your text, the order of consonant phonemes in each word must follow the [place of articulation](https://en.wikipedia.org/wiki/English_phonology); start with phonemes articulated at the back of the mouth, end with phonemes articulated at the front. For instance, "good" starts at the velum and ends at the alveolar ridge. You can follow the [table in this article](https://en.wikipedia.org/wiki/English_phonology#Consonants) from right to left.
* [ ] Nov. 6: **#mirrorlipo** -- You may only use letters that have at least one axis of symmetry.
* [ ] Nov. 7: **#interleaflipo** -- Reading every other word in your text must still yield a full, valid text.
* [ ] Nov. 8: **#recyclipo** -- The same set of letters must appear in each sentence of your text. 
* [ ] Nov. 9: **#quintilipo** -- All words in your text must contain at least 5 letters.
* [ ] Nov. 10: **#vowelringlipo** -- The order in which the vowels a, e, i, o, u appear (across words) must stay the same throughout the text. You can pick the order of your vowels.
* [ ] Nov. 11: **#monotonelipo** -- All words in your text must have the same length.
* [ ] Nov. 12: **#frankensteinlipo** -- Your text must contain the verbs from another, published text, in the order of appearance, but the substantives of yet another, published text, in the order of appearance; e.g., the verbs from _Pride and Prejudice_, but the nouns of _The Hobbit_. Words that are not verbs or nouns can be inserted freely.
* [ ] Nov. 13: **#aphasialipo** -- Your text may not contain any meaningful substantives; in case you want to use a substantive, you must use a placeholder such as "thing", "stuff", or "whatchamacallit".
* [ ] Nov. 14: **#upgolipo** -- You may only use the [ten hundred most used words in English](http://splasho.com/upgoer5/).
* [ ] Nov. 15: **#piphilolipo** -- Your text must contain words whose [length corresponds to the digits of pi](https://en.wikipedia.org/wiki/Piphilology), in [order](https://en.wikipedia.org/wiki/Pilish).
* [ ] Nov. 16: **#dashakumaracharitalipo** -- Your text [may not contain](https://en.wikipedia.org/wiki/Dashakumaracharita) any [labial consonants](https://en.wikipedia.org/wiki/Labial_consonant) ([m], [p], [b], [f], [v]).
* [ ] Nov. 17: **#panphonlipo** -- Write a text that contains [each phoneme of English](https://en.wikipedia.org/wiki/English_phonology) once. The shorter your text is, the better.
* [ ] Nov. 18: **#emojlipo** -- You may only use substantives if there is an emoji for them. Write the emoji instead of the word.
* [ ] Nov. 19: **#acrostilipo** -- The beginning letters of the words in your text must spell out another text, word, or phrase.
* [ ] Nov. 20: **#abstractlipo** -- You may not use any concrete nouns; all substantives you use must refer to abstract concepts.
* [ ] Nov. 21: **#hapaxlipo** -- Pick a published text and determine its [hapax legomena](https://en.wikipedia.org/wiki/Hapax_legomenon). Your own text may only consist of the original text's hapaxes.
* [ ] Nov. 22: **#rhopalipo** -- Each word in your text must be [longer than the previous word](https://www.merriam-webster.com/dictionary/rhopalic) by one syllable.
* [ ] Nov. 23: **#epanodoslipo** -- The second sentence of your text must [repeat one word from the first sentence](https://en.wikipedia.org/wiki/Epanodos). The next sentence must repeat that word and an additional word from the second sentence. Continue as long as you like.
* [ ] Nov. 24: **#selfsustainlipo** -- You may only use letters and characters that occur in your own full name and title.
* [ ] Nov. 25: **#dactylipo** -- Your entire text must consist of a row of [dactyls](https://en.wikipedia.org/wiki/Dactyl_(poetry)).
* [ ] Nov. 26: **#eprimelipo** -- Your text may contain [absolutely no forms of the verb "to be"](https://en.wikipedia.org/wiki/E-Prime), neither as a full verb nor as an auxiliary verb.
* [ ] Nov. 27: **#snpvplipo** -- Your entire text must consist of sentences of the form [S -> NP VP](https://en.wikipedia.org/wiki/Phrase_structure_rules), that is, sentences must conform to the structure of sentences like "John walks" or "John loves Mary".
* [ ] Nov. 28: **#anglipo** -- Your text may [only](https://en.wikipedia.org/wiki/Linguistic_purism_in_English) contain words of [Anglo-Saxon origin](https://en.wikipedia.org/wiki/List_of_English_words_of_Anglo-Saxon_origin).
* [ ] Nov. 29: **#ouroborolipo** -- Each sentence in your text must end with the same word with which it begins.
* [ ] Nov. 30: **#traindenullepartlipo** -- Your text [may not contain any verbs](https://en.wikipedia.org/wiki/Le_Train_de_Nulle_Part).

**Comment by LuRsT** [comment]
Will you be doing one entry per challenge? :open_mouth:

**Comment by hugovk** [comment]
No, I won't have time for that, but some will be double entries! Like 50,000 Woofs can do for both #avoidlipo and #monosyllablipo :)

**Comment by hugovk** [comment]
_A CATALOGUE OF PHARMACEUTICAL DRUG NAMES_

https://github.com/NaNoGenMo/2018/issues/95
